skgstat/binning.py

```diff
@@ -57,8 +58,8 @@ def binify_even_width(X, N=10, w=None, dm=None, maxlag=None, **kwargs):
             print('Warning! w = %d is ignored because N is already given' % w)
         w = maxval / N
 
-    # binubound = np.cumsum(np.ones(N) * w)
-    binubound = np.linspace(w, N * w, N)
+    # TODO replace with: binubound = np.linspace(w, N * w, N)
+    binubound = np.cumsum(np.ones(N) * w)
```

Comment:
Correct me if I'm wrong, but this is the old definition of the bin widths, right? Therefore it should be replaced by the removed line in order to store the bins upper bounds in binubound again.